concourse/tasks/update-ecs-service.sh

```diff
@@ -30,22 +30,30 @@ aws ecs update-service \
   --task-definition "$new_task_definition_arn" \
   --region "$AWS_REGION" > /dev/null
 
-# Get the ID of the latest task so we can stream its logs.
-# ecs-cli just wants the numeric ID of the task, not the whole ARN.
-# So we have to use `awk` to fish that out.
-task_id=$(aws ecs list-tasks \
+# Get the ID of the latest task.
+task_arn=$(aws ecs list-tasks \
   --cluster "$CLUSTER" \
   --service-name "$ECS_SERVICE" \
   --region "$AWS_REGION" \
   --query 'taskArns | [0]' \
-  --output text \
-  | awk -F / '{print $NF}'
+  --output text
 )
-echo "Task ID: $task_id"
+
+echo "Task ARN: $task_arn"
```

Comment:
This log message could be a bit confusing/strange as there would normally be more than one task. This prints the Task ARN of one of the tasks.

Reply:
So the previous definition of _update-csv-service.sh_ was only concerned with a single task, and when I tested this against my cluster it did not occur to me that we would have multiple tasks per service. In the old way of doing things, we would only be monitoring a single task via CloudWatch Logs during the deploy; in the case of multiple tasks, am I assuming that we want to monitor each?